command.go

```diff
@@ -24,18 +24,22 @@ var (
 	_ Cmder = (*StringIntMapCmd)(nil)
 	_ Cmder = (*ZSliceCmd)(nil)
 	_ Cmder = (*ScanCmd)(nil)
+	_ Cmder = (*ClusterSlotCmd)(nil)
 )
 
 type Cmder interface {
 	args() []string
 	parseReply(*bufio.Reader) error
 	setErr(error)
+	reset()
 
 	writeTimeout() *time.Duration
 	readTimeout() *time.Duration
+	firstKey() string
 
 	Err() error
 	String() string
+	SetKeyArgPos(int)
```

Comment:
OK, but I really think users should have option to set cluster keys for custom commands